DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/SearchableDelegateGroupViewModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+
+    public class SearchableDelegateGroupViewModel
+    {
+        private const string Yes = "Yes";
+        private const string No = "No";
+
+        public SearchableDelegateGroupViewModel(Group group)
+        {
+            Id = group.GroupId;
+            Name = group.GroupLabel;
+            Description = group.GroupDescription;
+            DelegateCount = group.DelegateCount;
+            CourseCount = group.CoursesCount;
+            LinkedToField = group.LinkedToField;
+            LinkedField = group.LinkedToFieldName;
+            AddedBy = $"{group.AddedByFirstName} {group.AddedByLastName}";

```

Comment:
Wasn't sure what we'd need later, figured it might be better to do it c# side just in case. Don't think it makes any difference at all where it happens though.